lib/cron_parser.dart

```diff
@@ -6,8 +6,16 @@ import 'package:timezone/standalone.dart';
 import 'package:timezone/timezone.dart';
 
 abstract class HasNext<E> {
+  /// Find next suitable date
   E next();
+}
```

Comment:
@rbubke  Because `current()` moved into `CronIterator`, is this PR a breaking change. I added a note in the changelog: 61303021656febb37420b4bb647991becb44c103